view-user/src/components/common/BackIcon/BackIcon.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { IconFramework } from "@/components/common";
+import { TiArrowBack } from "react-icons/ti";
+import { useRouter } from "next/router";
+
+const BackIcon = () => {
+  const router = useRouter();
+
+  const handleClick = () => {
+    if (typeof window !== "undefined") {
+      router.push("/");
```

Comment:
コンポーネント化しているため、特定のページにリダイレクトさせるよりも、

- パスをコンポーネントのプロパティとして受け取るようにする。
- 履歴を１つ戻す。

などの方が良い気がします。

```suggestion
      router.back();
```